app/assets/javascripts/models.coffee

```diff
@@ -2,17 +2,27 @@ exports.bindModelChooser = (container, selectionChanged) ->
 
   PUBLIC_PATH_SEGMENT_LENGTH = "public/".length
 
-  adjustModelName = (modelName) ->
+  adjustModelPath = (modelName) ->
     modelName.substring(PUBLIC_PATH_SEGMENT_LENGTH, modelName.length)
 
+  modelDisplayName = (modelName) ->
+    adjustedModelPath = adjustModelPath(modelName)
+    if adjustedModelPath.substring(0, 10) == "modelslib/"
```

Comment:
Unfortunately, support for that is first being added in ES6.  The sake of compatibility, our project targets ES5.